core/src/avm2/playerglobal/README.md

```diff
@@ -0,0 +1,49 @@
+# Ruffle playerglobal
```

Comment:
I think that the `playerglobal` directory doesn't have to be separate anymore. How about moving the `.as` files directly into `core/src/avm2/globals/`? This way we'll not have a mirrored file tree while migrating from Rust to ActionScript (hopefully a typical migration will be just a matter of renaming a `.rs` file to an `.as`).